notifications.bs

```diff
@@ -102,15 +102,24 @@ or vibration pattern that is not otherwise accessible to the end user.
 
 <p>A <a lt="concept notification">notification</a> has an associated list of
 zero or more <dfn>actions</dfn>. Each action has an associated
-<dfn lt="action title">title</dfn> and <dfn>action name</dfn>. Users may
-activate actions, as alternatives to activating the notification itself. The
-user agent must determine the <dfn>maximum number of actions</dfn> supported,
-within the constraints of the notification platform.
+<dfn lt="action title">title</dfn> and <dfn>action name</dfn> and <em>can</em>
+have an associated <dfn>action icon URL</dfn> and
+<dfn>action icon resource</dfn>. Users may activate actions, as alternatives to
+activating the notification itself. The user agent must determine the
+<dfn>maximum number of actions</dfn> supported, within the constraints of the
+notification platform.
 
 <p class=note>Since display of actions is platform-dependent, developers are
 encouraged to make sure that any action a user can invoke from a notification is
 also available within the web application.
 
+<p class="note no-backref">Some platforms may modify an
+<a>action icon resource</a> to better match the platform's visual style before
+displaying it to the user, for example by rounding the corners or painting it in
+a specific color. It is recommended to use an icon that handles such cases
+gracefully and does not lose important information through e.g. loss of color or
```

Comment:
While [this](http://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e) and [this](http://english.stackexchange.com/questions/7946/use-of-e-g-are-parentheses-necessary) page explain how this is recommended by some style guides, primarily American ones, it is not usually required. If anything, a comma after is more commonly recommended, and one could even use both one before and after.

 Personally, I find commas look unappealing in combination with the periods, and would prefer not to add any.